src/components/Dissolution/AssociationDetails.vue

```diff
@@ -165,9 +165,8 @@ export default class AssociationDetails extends Mixins(CommonMixin, EnumMixin, D
 
   /** The business start date. */
   get businessStartDate (): string {
-    // SB TODO: get data from object and pass here
-    // mostly it will be same as foundingDate
-    return this.yyyyMmDdToPacificDate(new Date().toString(), true)
+    // it will be same as foundingDate
+    return this.dateToPacificDate(new Date(this.getBusinessFoundingDate), true)
```

Comment:
DO NOT use `new Date()` as it will construct the date based on the user's browser's current time and timezone (which may be wrong or spoofed).

Instead, use one of the Date Mixin functions to convert Business Founding Date to a JS Date, then use dateToPacificDate().